pkg/instrumentation/sdk.go

```diff
@@ -58,6 +58,12 @@ func inject(logger logr.Logger, insts languageInstrumentations, ns corev1.Namesp
 		pod = injectCommonSDKConfig(otelinst, ns, pod)
 		pod = injectNodeJSSDK(logger, otelinst.Spec.NodeJS, pod)
 	}
+	if insts.Python != nil {
+		otelinst := *insts.Python
+		logger.V(1).Info("injecting instrumentation into pod", "otelinst-namespace", otelinst.Namespace, "otelinst-name", otelinst.Name)
```

Comment:
it would be good to include python in the log message

Reply:
Thanks, fixed